sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkBatchClient.cs

```diff
@@ -1,41 +1,243 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// <auto-generated/>
```

Comment:
This marker turns off quite a few of our azure sdk analyzers so we should always remove this outside the `.../Generated/...` folder.